.evergreen/run-tests.sh

```diff
@@ -157,8 +138,9 @@ if [ -n "$TEST_ENCRYPTION" ] || [ -n "$TEST_FLE_AZURE_AUTO" ] || [ -n "$TEST_FLE
     export PYMONGOCRYPT_LIB
 
     # TODO: Test with 'pip install pymongocrypt'
-    git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
-    python -m pip install --prefer-binary -r .evergreen/test-encryption-requirements.txt
+    if [ ! -d "libmongocrypt_git" ]; then
+        git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
```

Comment:
Do we need to do this if we already install libmongocrypt above?

Reply:
It shouldn't be